subscriptions/tasks.py

```diff
@@ -174,28 +174,35 @@ def run(self, subscription_id, resend=None, **kwargs):
         """
         log = self.get_logger(**kwargs)
 
+        context = {
+            "subscription_id": subscription_id,
+            "resend": resend
+        }
+
         log.info("Loading Subscription")
         try:
             subscription = Subscription.objects.get(id=subscription_id)
         except Subscription.DoesNotExist:
             logger.error('Could not load subscription <%s>' % subscription_id,
                          exc_info=True)
-            return
+            context['error'] = ('Could not load subscription <%s>' %
+                                subscription_id)
+            return context
```

Comment:
Logging and exiting doesn't seem like a good strategy for this.

This was necessary since the ID used to come from an API endpoint, so if another application pinged that endpoint with an invalid ID, then we wouldn't want a sentry error on our side, since it's not our problem.

This task is now only called by our internal methods, so if we are given a subscription ID that doesn't exist, then something is very wrong, and we should probably get an error in Sentry.

I think we can just remove the try/except for the subscription lookup, because if we cannot find it, then we want this to blow up and show a stack track in Sentry.